services/cadexsoft/app/main.py

```
@@ -0,0 +1,196 @@
+    )
+
+
+# ---------------------------------------------------------------------------
+# Analysis endpoints — all accept JSON (AnalyzeRequest) or multipart upload.
+# ---------------------------------------------------------------------------
+
+def _analyze_route(kind: AnalysisKind):
+    async def handler(
+        request: Request,
+        body: Optional[AnalyzeRequest] = None,
+        upload: Optional[UploadFile] = File(default=None),
```

Ruff (0.15.10)

[warning] Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)